pytensor/configdefaults.py

```diff
@@ -1437,12 +1437,16 @@ def add_caching_dir_configvars():
 add_numba_configvars()
 
 # TODO: `gcc_version_str` is used by other modules.. Should it become an immutable config var?
-try:
-    p_out = output_subprocess_Popen([config.cxx, "-dumpversion"])
-    gcc_version_str = p_out[0].strip().decode()
-except OSError:
-    # Typically means gcc cannot be found.
+if config.cxx != "":
+    try:
+        p_out = output_subprocess_Popen([config.cxx, "-dumpversion"])
+        gcc_version_str = p_out[0].strip().decode()
+    except OSError:
```

Comment:
an alternative fix would be to catch everything instead of just `OSError`, but the proposed change seemed narrower/safer.